### PR TITLE
doc: Moving oberon changeloge to doc

### PR DESCRIPTION
**Pull request by mswarowsky**
Part of centraliting the place for the changelogs of the crypto libaries

### CODE REVIEW

File: crypto/README.rst [hunk 1]
```diff
@@ -11,8 +11,8 @@ Crypto Libraries
    doc/nrf_cc310_platform
    doc/nrf_cc310_mbedcrypto
    doc/nrf_oberon
-   nrf_cc310_platform/CHANGELOG.rst
-   nrf_cc310_bl/CHANGELOG
-   nrf_cc310_mbedcrypto/CHANGELOG
-   nrf_oberon/CHANGELOG
+   doc/CHANGELOG_cc310_bl
```
Comment (melwee01) [inline crypto/README.rst:14]: Does it matter that the previous link specified the RST file extension and the new one doesn't?

Comment (mswarowsky) [reply crypto/README.rst:14]: I don't think so as all the other files don't have it I aligned it with the rest.

**Review by SebastianBoe — APPROVED** [review]

**Review by mia-ko — APPROVED** [review]

**Comment by joerchan** [comment]
This looks good, but I on looking at the changelog titles I'm noticing a discrepancy in naming and I'm wondering of you could fix that also as part of this PR.
image

Could you update to these?
Changelog - nrf_cc3xx_bl
Changelog - nrf_oberon

**Comment by mia-ko** [comment]
> This looks good, but I on looking at the changelog titles I'm noticing a discrepancy in naming and I'm wondering of you could fix that also as part of this PR. image
> 
> Could you update to these? Changelog - nrf_cc3xx_bl Changelog - nrf_oberon

Ah, I agree with Joakim here, it should definitely be aligned. But it has to be in a different PR it seems as this was just merged.
Joakim, it says nrf_cc310_bl crypto library, do you want the changelog to be changed to Changelog - nrf_cc3xx_bl and not nrf_cc310_bl? And if nrf_cc3xx_bl, does that mean that the library should be renamed as well? Just checking :)

**Comment by joerchan** [comment]
> do you want the changelog to be changed to Changelog - nrf_cc3xx_bl

No, that was just a mistake, no renaming.

**Pull request merged**